coding-concepts.js

```diff
@@ -8,36 +8,49 @@
 // b) Verify and explain:
+// Output: [
+//   'C', 'h', 'a', 'r',
+//   'l', 'i', 'e', ' ',
+//   '2', '0', '2', '3'
+// ]
+// Explain: The cohort string variable is split into each index of the value by the .split("") method. The .split() method is called upon the cohort variable, passing an empty string `""` as a seperator argument. When an empty string is used as the sperator argument each char within the string is seperated, making each char a single element, including empty spaces. 
 
 // --------------------2) What will this log?
 
 const greeter = (name) => {
   `Hello, ${name}!`
 }
-// console.log(greeter("LEARN Student"))
+console.log(greeter("LEARN Student"))
 
-// a) Your answer:
+// a) Your answer: I dont think it will log anything. Ther is no return to this fuction, therfore it probably will come back undefined. 
 // b) Verify and explain:
+//Output: undefined
+//Why? the greeter () function is not defined without a return statement. When the argument "LEARN Strudent" is passed into the function call as an argument, the program doesnt understand what to output without the return statement following.
 
 // --------------------3) What will this log?
 
 const multipliedByTwo = [4, 5, 6, 7, 8].map((number) => number * 2)
-// console.log(multipliedByTwo)
+console.log(multipliedByTwo)
 
-// a) Your answer:
-// b) Verify and explain:
+
+// a) Your answer:[8,10,12,14,16], because the function maps out the array and multiplies each index by 2
+// b) Verify and explain: 
+// Output: [ 8, 10, 12, 14, 16 ]
+// Why? The .map() function iterates over each value in the array, and the ((value)=> value___) function manipulates each value by multiplying the values by 2. The codes is defined by the function call `multipliedByTwo`, so when called upon with the console log, the function is executed.
 
 // --------------------4) What will this log?
 
 const onlyOdds = [11, 12, 13, 14, 15].filter((number) => number % 2 !== 0)
-// console.log(onlyOdds)
+console.log(onlyOdds)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: it wont long the odds because the finding out would be stricly equal to(===). i think it will either leave the even number or come back with an error. Then again I dont really understand what the !(not)==(loosly equal) means, like why couldnt they just use `!=`? 
+// b) Verify and explain: 
+//Output: [ 11, 13, 15 ]
+//Why? The defining element of this function is `onlyOdds`. When called upon in the console.log(), the function is execited. Taking the array and applying .filter() method creates a new array based upon the conditions implemented by the arrow function ((number) => number % 2 !== 0); the ending statement `number % 2 !== 0` being the conditional. The modulo operator (%) checks if the remainder is not equal to 0. If the remainder is not, then the value is odd and is output in the new array. 
```

Comment:
I would review your vocabulary; I know what you are trying to say but you are not saying it correctly! `onlyOdds` is not a function, it is a variable. Also, you said the "defining element of this function" is onlyOdds, what function are you referring to?